models/evms/evms_transfers.sql

```diff
@@ -0,0 +1,49 @@
+{{ config(
+        tags = ['dunesql'],
+        alias = alias('transfers'),
```

Comment:
just double checking here --
you have alias `transfers` but the source tables below are `blocks`? is that correct?

Reply:
that file isn't supposed to be here, just deleted it